project/Dependencies.scala

```diff
@@ -26,4 +26,9 @@ object Dependencies {
 
   val play = mainPlay ++ testPlay
 
+  def pickVersion(scalaVersion: String, ver211: String, ver212: String) = CrossVersion.partialVersion(scalaVersion) match {
```

Comment:
Right... silly me. Funny thing - I already did something along these lines but forgot now: https://github.com/pdolega/blueprints-scala/blob/master/build.sbt#L35

Getting old... (and will fix :)